_events/2020-encampment-work-party.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Encampment Work Party
+title: Encampment Work Party
+date: 2020-01-24
+startDate: '2020-01-25 08:30'
+endDate: '2020-01-25 12:00'
+pinned: true
+location:
+  name: Kernville Park Recreation Area
+  addressLocality: Kernville
+  addressRegion: CA
+  postalCode: 93238
+  url: 'https://goo.gl/maps/Y8tsP1htneWGUSnV8'
+  geo:
+    latitude: 35.76067
+    longitude: -118.424131
+image: 'https://i.imgur.com/mhfGbqc.jpg'
+imgur: 'https://i.imgur.com/mhfGbqc'
+description: 'Community welcome to help Encampment reenactors put some finishing touches on the entire area.'
+tags:
+- events
+- kern valley
+- kern river valley
+- kernville
+- whiskey flat
+- whiskey flat days
```

Comment:
Should definitely include whiskey flat Encampment. Both with and without the "whiskey flat" part.

Think of tags as what searches this is relevant to.